.ci/jobDSL/jobs/apm-ci/apm-shared/docker-images/build_docker_images.groovy

```diff
@@ -85,6 +85,126 @@ def dockerImages = [
     build_script: "docker build --force-rm -t ${registry}/${prefix}/functional-opbeans:latest functional-opbeans",
     push_script: "docker push ${registry}/${prefix}/functional-opbeans:latest",
     push: true
+  ],
+  [
+    name: 'flakey',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "apps/automation/jenkins-toolbox"
+  ],
+  [
+    name: 'flakeyv2',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "apps/automation/flaky-test-analyzer"
+  ],
+  [
+    name: 'build-analyzer',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "apps/automation/build-analyzer"
+  ],
+  [
+    name: 'rebuild-analyzer',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "apps/automation/rebuild-analyzer"
+  ],
+  [
+    name: 'integrations-test-reporter',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "apps/automation/integrations/reporter"
+  ],
+  [
+    name: 'slack-bridge-hey-apm',
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: 'latest',
+    push: true,
+    folder: "tools/report-bridge"
+  ],
+  [
+    name: 'obs-jenkins-heartbeat',
+    repo: "https://github.com/elastic/observability-robots.git",
+    tag: 'latest',
+    push: true,
+    prepare_script: '''
+      cd ${BASE_DIR}/${params.folder}
+      pip3 install pyyaml
+      python3 ./generate_heartbeat_configs.py
+    ''',
+    folder: "apps/beats/heartbeat"
+  ],
+  [
+    name: "bandstand",
+    repo: 'https://github.com/elastic/observability-dev',
+    tag: "latest",
+    folder: "apps/automation/bandstand",
+    push: true
+  ],
+  [
+    name: "azure-vm-tools",
+    repo: 'https://github.com/elastic/azure-vm-extension',
+    tag: "latest",
+    folder: ".ci/docker/azure-vm-tools",
+    push: true
+  ],
+  [
+    name: 'picklesdoc',
+    repo: 'https://github.com/elastic/observability-robots.git',
+    tag: 'latest',
+    build_script: 'make build',
+    push_script: 'make push',
+    push: true,
+    folder: "apps/pickles"
+  ],
+  [
+    name: 'test-plans',
+    repo: 'https://github.com/elastic/observability-robots.git',
+    tag: 'latest',
+    build_script: 'make build',
+    push_script: 'make push',
+    push: true,
+    folder: "apps/test-plans"
+  ],
+  [
+    name: "apm-integration-testing",
+    repo: 'https://github.com/elastic/apm-integration-testing.git',
+    tag: "daily",
+    push: true
+  ],
+  [
+    name: "apm-integration-testing-aux",
```

Comment:
What's this?

Reply:
https://github.com/elastic/apm-pipeline-library/pull/1534/files#diff-596057068aaa2ba87378d9db4a5f48e3dd8bb52c02accb89b04331587aa09e4cL423-L431


maybe a comment about the aim of this ?

Reply:
It compiles all the APM ITs Docker images